BankingSystem.DataAccess/Data/Context.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace BankingSystem.DataAccess.Data
+{
+    public class Context:DbContext
```

Comment:
Rename class name to BankingSystemDbContext

Make sure to format file before comitting (with needed spaces)